src/serlo-editor/math/editor.tsx

```diff
@@ -89,7 +93,19 @@ export function MathEditor(props: MathEditorProps) {
 
   const { visual, readOnly, state, disableBlock } = props
 
-  const useVisualEditor = visual && !hasError
+  useHotkeys(
+    Key.Escape,
+    (event) => {
+      event.preventDefault()
+      // close overlay
+      props.onMoveOutRight()
```

Comment:
we already destructure props above, maybe add it here and use `onMoveOutRight()`?